## Request Constants
int8 GET = 1
int8 PUT = 2
int8 POST = 3
int8 DELETE = 4
# Request Fields
int8 request_type